types/offscreencanvas/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for non-npm package offscreencanvas-browser 2019.3
+// Project: https://html.spec.whatwg.org/multipage/canvas.html#the-offscreencanvas-interface
+// Definitions by: Klaus Reimer <https://github.com/kayahr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 3.1
+
+declare global {
```

Comment:
if you put these declarations at the top-level, they should work the same. I don't think you need the `declare global { ... }; export { }` boilerplate.

Reply:
Fixed